editor/code_editor.cpp

```diff
@@ -757,6 +759,8 @@ FindReplaceBar::FindReplaceBar() {
 
 /*** CODE EDITOR ****/
 
+const float CodeTextEditor::ZOOM_FACTOR_PRESETS[7] = { 0.25f, 0.5f, 0.75f, 1.0f, 1.5f, 2.0f, 3.0f };
```

Comment:
I think you can add `static` before `const` here so you don't need to declare it in the header.